packages/dds/shared-object-base/src/test/serializer.spec.ts

```diff
@@ -37,20 +37,10 @@ describe("FluidSerializer", () => {
 
 	// Type tests for FluidSerializableReadOnly
 	{
-		const allows: FluidSerializableReadOnly[] = [
-			[],
-			{},
-			5,
-			null,
-			"",
-			{ x: 5 },
-			[[]],
-			true,
-			new MockHandle(1),
-		];
+		const use = (x: FluidSerializableReadOnly) => undefined;
+		[[], {}, 5, null, "", { x: 5 }, [[]], true, new MockHandle(1)].map(use);
```

Comment:
I'd throw a comment here
```suggestion
		// These types of values are allowed
		[[], {}, 5, null, "", { x: 5 }, [[]], true, new MockHandle(1)].map(use);
```